src/control/control.cpp

```diff
@@ -148,11 +148,12 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
             auto pControl = it.value().lock();
             if (pControl) {
                 // Control object already exists
-                VERIFY_OR_DEBUG_ASSERT(!pCreatorCO) {
+                if (pCreatorCO) {
                     qWarning()
                             << "ControlObject"
                             << key.group << key.item
                             << "already created";
+                    DEBUG_ASSERT(false);
```

Comment:
The condition is out of sight here. Please use something like
```suggestion
                    DEBUG_ASSERT(!"Tried to create Control Object that already exists.");
```